dds_web/api/user.py

```diff
@@ -213,6 +210,7 @@ def add_user_to_project(existing_user, project, role):
                     owner=owner,
                 )
             )
+            share_project_private_key_with_user(auth.current_user(), existing_user, project)
```

Comment:
This should also be called if the researchuser has an account but is not connected to the project yet

Reply:
As I understand this function is executed when a ownership change happens as well. In that case there is no new project keys to share as the user already has access to it. It is only a title change of some sort not functional. If the function is not executed due to ownership change, it is to add an existing user to the project so line 213 will execute.